src/ItemFieldSet.tsx

```diff
@@ -70,19 +70,24 @@ export class ItemFieldSet extends React.Component<IItemFieldSetProps, IItemField
     </IconButton>);
     const itemsArray = this.state.items.map((item: IItemInternal, itemIndex: number) => {
       let minusBtn = null;
+      let autoFocus = false;
       if (itemIndex > 0) {
         minusBtn = (
           <IconButton aria-label="Remove item" onClick={this.remove(itemIndex)}>
             <RemoveCircleIcon style={{ height: '24px', width: '24px' }} />
           </IconButton>)
       }
+      if (itemIndex === 0) {
+        autoFocus = true;
+      }
       return (<div key={itemIndex}>
         <Card key={itemIndex}>
           <CardContent>
             <p>Describe Item Consumed</p>
             {minusBtn}
             <div>
               <TextField type="text"
+                autoFocus={autoFocus}
```

Comment:
no need to create a variable, replace this line with:
autoFocus={itemIndex === 0}